admin/load.php

```diff
@@ -13,376 +13,56 @@
  * Adds the modules page to the Settings menu.
  *
  * @since 1.0.0
+ * @since n.e.x.t Renamed to perflab_add_features_page().
  */
-function perflab_add_modules_page() {
-	// Don't add a page if active modules are controlled programmatically.
-	if ( has_filter( 'perflab_active_modules' ) ) {
-		return false;
-	}
-
+function perflab_add_features_page() {
 	$hook_suffix = add_options_page(
-		__( 'Performance Modules', 'performance-lab' ),
+		__( 'Performance Features', 'performance-lab' ),
 		__( 'Performance', 'performance-lab' ),
 		'manage_options',
 		PERFLAB_MODULES_SCREEN,
-		'perflab_render_modules_page'
+		'perflab_render_settings_page'
 	);
 
 	// Add the following hooks only if the screen was successfully added.
 	if ( false !== $hook_suffix ) {
-		add_action( "load-{$hook_suffix}", 'perflab_load_modules_page', 10, 0 );
+		add_action( "load-{$hook_suffix}", 'perflab_load_settings_page', 10, 0 );
 		add_filter( 'plugin_action_links_' . plugin_basename( PERFLAB_MAIN_FILE ), 'perflab_plugin_action_links_add_settings' );
 	}
 
 	return $hook_suffix;
 }
-add_action( 'admin_menu', 'perflab_add_modules_page' );
+add_action( 'admin_menu', 'perflab_add_features_page' );
 
 /**
  * Initializes settings sections and fields for the modules page.
  *
  * @since 1.0.0
- *
- * @global array $wp_settings_sections Registered WordPress settings sections.
- *
- * @param array|null $modules     Associative array of available module data, keyed by module slug. By default, this
- *                                will rely on {@see perflab_get_modules()}.
- * @param array|null $focus_areas Associative array of focus area data, keyed by focus area slug. By default, this will
- *                                rely on {@see perflab_get_focus_areas()}.
+ * @since n.e.x.t Renamed to perflab_load_settings_page(), and the
+ *                $module and $hook_suffix parameters were removed.
  */
-function perflab_load_modules_page( $modules = null, $focus_areas = null ) {
-	global $wp_settings_sections;
-
+function perflab_load_settings_page() {
```

Comment:
Since you went with `features_page` in the other function above, please align that here as well.